src/gen_rules.ml

```diff
@@ -1430,16 +1503,19 @@ module Gen(P : Params) = struct
       guess_modules ~dir:src_dir
         ~files:(Lazy.force files))
     in
+    let merlin_deps = ref [] in
+    let add_merlin_dep x = merlin_deps := x :: !merlin_deps in
```

Comment:
Good point.  I just pushed an update and was curious if the new changes were more idiomatic